### ISSUE TITLE
Re-enable Mekanism Transporters (Cables)

### ISSUE THREAD

**Issue by Fireztonez** [Darkosto/SkyFactory4Issues#458]
## Context
I know Mekanism Transports have been removed from the pack because of lags and bugs with it on old version, and this is totally understanable... But A good team of dev have work together to optimized a lot the codes and pipes (On Mekanica), and all that work have been recently merge with the real Mekanism (With the 9.7 version). I have test and look, and now, sincerly Mekanism Transporter work really great and is pretty optimized... I think putting that back should be nice, at view of all the work done to make Mekanism a really more optimized mod...

And, for having test both on my modpack, Item Pipe and items transfer node from Cyclic is 50x more laggy than mekanism pipe, but is still present on the pack (I already have report that and the dev have start optimized that a lot!!).

no worry, if you take decision to let that disabled this is ok, after all the pack is already out, so, if you whant to go back and change this or not, and test thing with it, this is clearly your choice. But I see a lot of person asking why this is not in the pack and things like that, and, with the new version of meka, I don't see really a good reason.

## Config/Script Details
* Link: Mekanism 9.7.0 Changelogs: https://minecraft.curseforge.com/projects/mekanism/files/2698323
You can also look the new team working with aidancbrady and join the new Mekanism discord, dev is very very active!

And, thank's Darkosto and all other dev working on SF4, for making that a good modpack like it is!
Fireztonez

**Comment by Alleluid** [comment]
I agree with this, the transporters have some unique mechanics. If the reason they are out is for performance, then why is Cyclic in the pack? It's...not especially friendly to servers.

**Comment by Fireztonez** [comment]
@Alleluid Yeah, Cyclic now is not really server friendly... I try to help the best I can the dev question to optimized is mod, but i'm not a programmer, so I can't do really more than making profiling, report and suggestion question of optimizations...

I work on a modpack, made specially for running in server, so, I look closely what is the worst thing for server, and I have re-enabled Mekanism Logistical Transporter, because now it is pretty mutch optimized like Thermal Dynamics and EnderIO Conduit... But, Cyclic Item Cable is removed from the pack...

Anyways, the final decision have to be made by the SF4 dev.
